packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -17,7 +18,7 @@ export const styles = css`
         inline-size: auto;
         overflow: auto;
         block-size: 100%;
-        min-block-size: 36px;
+        min-block-size: ${bodyFontLineHeight};
```

Comment:
Do we really need to specify a minimum height in nimble? If the component is completely empty why should it use any space? It would be like an empty `div` or a `span`. Can we just remove the min size altogether?

Reply:
Yes, it makes sense not to have a minimum height for the viewer component and so I removed it.